### ISSUE TITLE
Using incorrect version of url_launcher

### ISSUE THREAD

**Issue by DanMossa** [supabase-community/supabase-flutter#112]
Hello!

I've updated my supabase_flutter package to `supabase_flutter: ^0.3.1`

Now when I go to run my app I get the following error:

```
The method 'launchUrl' isn't defined for the class 'GoTrueClient'.
 - 'GoTrueClient' is from 'package:gotrue/src/gotrue_client.dart' ('../../../../.pub-cache/hosted/pub.dartlang.org/gotrue-0.2.2/lib/src/gotrue_client.dart').
Try correcting the name to the name of an existing method, or defining a method named 'launchUrl'.
    final result = await launchUrl(url, webOnlyWindowName: '_self');
                         ^^^^^^^^^
```

And then inside of `lib/src/supabase_auth.dart`, `launchUrl` errors because it does not exist.

The current `pubspec.yaml` shows: `  url_launcher: ^6.0.10`
And this version of url_launcher does not have `launchUrl`.

**Comment by DanMossa** [comment]
Fix in #113

**Comment by dshukertjr** [comment]
Closing this as the fix has been merged.
